### ISSUE TITLE
Question about the train/test split

### ISSUE THREAD

**Issue by hxwork** [fnzhan/Illumination-Estimation#19]
Dear authors,

Thanks for opening source your code, and it is really helpful. I have a question about the train/test split of the Laval HDR dataset when I read the paper. It says that 19,556 training pairs are generated from 2,100 HDR panoramas, and 200 images are randomly selected as the test set. I am not sure if the 200 images are selected from the 2,100 HDR panoramas, or just from the 19,556 generated images. If they are from the 19,556 images, how to make sure the corresponding HDR panoramas of them are not in the training set or not seen during the training process?

Best regards,
Hao XU

**Comment by fnzhan** [comment]
Hi Hao Xu, as there is no common split of training/testing set for Laval HDR, I just randomly select 200 images from 19,556 training pairs. Thus, the test images and training images may be the same 3D scene but in different views. Of course, selecting test scenes from 2,100 HDR panoramas will be better for accurate evaluation.

**Comment by hxwork** [comment]
Got it. Thanks for your rapid reply.
